docs/source/nlp/tutorials/quickstarts/plot_multi_label_classification.py

```diff
@@ -0,0 +1,228 @@
+# tokenizers are very common.
+#
+# Integrity #2: Conflicting Labels
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Looking at the Conflicting Labels check result (in the “Didn’t Pass” tab) we can
+# see that there are 2 occurrences of duplicate samples that have different labels.
+# This may suggest a more severe labeling error in the dataset which we would want to explore further.
+#
+
+# %%
+# Train Test Validation
+# ---------------------
+# The next suite serves to validate our split and compare the two dataset. This suite is useful for when you already
+# decided about your train and test/validation splits, but before training the model itself. It is recommended to run
+# this suite during your model monitoring to verify the data hasn't change over time.
+#
+# For running the suite we need to split the data into train and test/validation sets. We'll use a predefined split
+# our data based on comment dates.
+
+from deepchecks.nlp.suites import train_test_validation
+
+train_ds, test_ds = just_dance_comment_analysis.load_data(data_format='TextData', as_train_test=True,
+                                                          include_embeddings=True, include_properties=True)
+train_test_validation(n_samples=1000).run(train_ds, test_ds, model_classes=class_names)
+
+# %%
+# Train Test Validation #1: Properties Drift
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Based on the different properties we have calculated for the dataset, we can now search for
+# properties whose distribution changes between the train and test datasets. Changes like this
+# are especially important to look for when monitoring your model over time, as data drift
+# is one of the top reasons why machine learning model’s performance degrades over time.
+#
+# In our case, we can see that the “% Special Characters”  and the "Formality" property have different distributions
+# between train and test. Drilling further into the results, we can see that the language of the comments in the
+# test set is much less formal and includes more special characters (possibly emojis?) than the train set.
+# Since this change is quite significant, we may want to consider adding more informal comments containing
+# special characters to the train set before training (or retraining) our model.
+#
+# Train Test Validation #2: Embedding Drift
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Similarly to the properties drift, we can also look for embedding drift between the train and test datasets.
+# The benefit of using embedding on top of the properties is that they are able to detect semantic changes in the data.
+#
+# In our case, we see that are significant semantic differences between the train and test sets. Specifically,
+# we can see some distinct segments that distinctly contain more
+# samples from the train dataset or more sample from the test dataset. By hovering over the segments we can see
+# text samples contained in each one and find the common ground.
```

Comment:
actually here the sentence I was missing in the other quickstart appears ("By hovering over..."). Just phrasing should be slightly improved